src/Tribe/Privacy.php

```diff
@@ -54,4 +56,176 @@ public function default_privacy_policy_content() {
 		return apply_filters( 'tribe_tickets_default_privacy_policy_content', $content );
 
 	}
+
+	/**
+	 * Register exporter for Tickets attendees saved data.
+	 *
+	 * @since TBD
+	 * @param $exporters
+	 *
+	 * @return array
+	 */
+	public function register_exporter( $exporters ) {
+		$exporters[] = array(
+			'exporter_friendly_name' => __( 'Event Ticket RSVP Atendee' ),
+			'callback'               => array( $this, 'rsvp_exporter' ),
+		);
+
+		$exporters[] = array(
+			'exporter_friendly_name' => __( 'Event Ticket TribeCommerce Atendee' ),
+			'callback'               => array( $this, 'tpp_exporter' ),
+		);
+
+		return $exporters;
+	}
+
+	/**
+	 * Exporter for Events Ticket RSVP Attendee
+	 *
+	 * @param     $email_address
+	 * @param int $page
+	 * @since     TBD
+	 *
+	 * @return array
+	 */
+	public function rsvp_exporter( $email_address, $page = 1 ) {
+		$number = 500; // Limit us to avoid timing out
+		$page   = (int) $page;
+
+		$export_items = array();
+
+		// Get the attendees RSVPs for the given email.
+		$rsvp_attendees = new WP_Query( array(
+			'post_type'      => Tribe__Tickets__RSVP::ATTENDEE_OBJECT,
+			'meta_key'       => '_tribe_rsvp_email',
+			'meta_value'     => $email_address,
+			'page'           => $page,
+			'limit'          => $number,
+			'orderby'        => 'ID',
+			'order'          => 'ASC',
+		) );
+
+		foreach ( $rsvp_attendees->posts as $attendee ) {
+
+			$item_id = "tribe_rsvp_attendees-{$attendee->ID}";
+
+			// Set our own group for RSVP attendees
+			$group_id = 'rsvp-attendees';
+
+			// Set a label for the group
+			$group_label = __( 'Event Tickets RSVP Attendee Data', 'event-tickets' );
+
+			$data = array();
+
+			$data[] = array(
+				'name'  => __( 'RSVP Title', 'event-tickets' ),
+				'value' => get_the_title( $attendee->ID ),
+			);
+
+			$data[] = array(
+				'name'  => __( 'Full Name', 'event-tickets' ),
+				'value' => get_post_meta( $attendee->ID, '_tribe_rsvp_full_name', true ),
+			);
+
+			$data[] = array(
+				'name'  => __( 'Email', 'event-tickets' ),
+				'value' => get_post_meta( $attendee->ID, '_tribe_rsvp_email', true ),
+			);
+
+			$data[] = array(
+				'name'  => __( 'Date', 'event-tickets' ),
+				'value' => $attendee->post_date,
+			);
+
+			$export_items[] = array(
+				'group_id'    => $group_id,
+				'group_label' => $group_label,
+				'item_id'     => $item_id,
+				'data'        => $data,
+			);
+		}
+
+		// Tell core if we have more comments to work on still
+		$done = count( $rsvp_attendees->posts ) < $number;
+
+		return array(
+			'data' => $export_items,
+			'done' => $done,
+		);
+	}
+
+	/**
+	 * Exporter for Events Ticket Tribe Commerce Attendee
+	 *
+	 * @param     $email_address
+	 * @param int $page
+	 * @since     TBD
+	 *
+	 * @return array
+	 */
+	public function tpp_exporter( $email_address, $page = 1 ) {
+		$number = 500; // Limit us to avoid timing out
```

Comment:
Do we need to add a way to make export more later? if so lets add a `@todo` in code too.

Reply:
No, the idea is that is done in batches, and that's the number of each batch. WordPress added this, so the request for retrieving personal data doesn't timeout (plugins and themes can hook in there.)